### PR TITLE
for testing - Restore isl files

### PR DESCRIPTION
**Pull request by cpauya**
### General

This fix is for https://github.com/learningequality/kolibri-installer-windows/issues/122

## Reviewer Guidance

Do not merge yet.

### CODE REVIEW

**Comment by mrpau-richard** [comment]
I tested the Spanish language from the built Windows installer in this [PR](https://github.com/learningequality/kolibri/pull/5814) and it display the strings properly

Screen Shot 2019-07-26 at 5 35 44 PM

**Comment by mrpau-julius** [comment]
I also tested the Spanish language from this [PR ](https://github.com/learningequality/kolibri/pull/5814). It displays properly.

sample1

sample2

display3

display4

File: windows/inno-compiler/Languages/Spanish.isl [hunk 1]
```diff
@@ -322,27 +322,27 @@ ShutdownBlockReasonUninstallingApp=Desinstalando %1.
...
-InstallPythonMsg=La versión 3.4+ de Python es necesaria para instalar Kolibri en Windows. ¿Desea instalar Python 3.4.3, antes de continuar con la instalación de Kolibri?
+UpgradeMsg=Hemos detectado una instalaci�n existente de Kolibri; �quieres actualizar?
+UpgradeDelMsg=Instalaci�n fresca borrar� todos sus datos existentes; �Esto es lo que realmente quieres hacer?
+InstallPythonMsg=La versi�n 3.4+ de Python es necesaria para instalar Kolibri en Windows. �Desea instalar Python 3.4.3, antes de continuar con la instalaci�n de Kolibri?
 InstallPythonErrMsg=Advertencia - Kolibri necesita Python para funcionar. Haga clic en Aceptar para volver e instalar Python, o cancelar para salir del instalador de Kolibri
 SetupKolibriErrMsg=Critical error. Dependencies have failed to install. Error Number:
-KolibriInstallFailed=Instalación de Kolibri ha fallado. Instalador de Kolibri encontró un errorinstallation process%n%n
-SetupWizardMsg=Asistente de instalación está copiando los archivos. Esto puede tomar un tiempo, por favor espere...
+KolibriInstallFailed=Something went wrong during Kolibri setup.%nAdditional information can be found in the log file:
```
Comment (indirectlylit) [inline windows/inno-compiler/Languages/Spanish.isl]: This one string looks like it got flipped from Spanish to English

**Comment by indirectlylit** [comment]
targeting hotfix to master and merging.

**Pull request merged**